---
layout: archive
title: "Mukta G. Palshikar"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

**GitHub: [Mukta Palshikar](https://github.com/muktapalshikar)**<br>
**Publications: [Google Scholar](https://scholar.google.com/citations?hl=en&user=wKCY16IAAAAJ&view_op=list_works&sortby=pubdate), [PubMed](https://pubmed.ncbi.nlm.nih.gov/?term=mukta+palshikar)**<br>

<span style="color:#154360">Education and Training</span>
======
* **Research Scientist (Jan 2026 - present)**
  - University of Nebraska, Lincoln (working remotely from Boston, MA).
  - Advisor: Tomas Helikar, PhD
  - Research focus: Developing computational approaches to the analysis of signaling networks, using integrated multi-omics datasets
* **Post-doctoral research fellow (Feb 2023 - Dec 2025)**
  - Harvard Medical School; Brigham and Women's Hospital, Boston, MA
  - Advisor: Ilya Korsunsky, PhD
  - Research focus: Spatial omics analysis and methods development to characterize cellular interactions in colorectal cancer and in interstitial lung disease. 

<span style="color:#154360">Education and Training</span>
======

* **PhD, Biophysics (Aug 2017 - Jan 2023)**
  - University of Rochester, NY
  - Advisor: Juilee Thakar, PhD
  - Dissertation: Executable models of signaling pathways built using omics data
* **Master of Science, Biophysics (2020)**
  - University of Rochester, NY

<!--  * **Master of Science, Bioinformatics (2015 - 2017)**
  - Savitribai Phule Pune University Pune, India
  - Advisor: Urmila Kulkarni-Kale, PhD & Mohan Kale, PhD
  - Thesis: Evaluation of Algorithms to Infer Boolean Networks From Time Series Data
* **Bachelor of Science, Microbiology & Chemistry 2011 - 2014**
  - Savitribai Phule Pune University Pune, India -->

<span style="color:#154360">Publications and Conference Presentations</span>
======

Preprints & In Preparation
------
1. Chen JH\*, **Palshikar MG\***, Chen H\*, ..., Hacohen N, Korsunsky I, Pelka K. Spatial analysis of human colorectal cancer identifies anti-tumor immunity hubs at epithelial/stromal interface as site enriched for effector and exhausted T cells. *In preparation*.
2. Tran M*, Gao C*, **Palshikar MG***, ..., Kim E, Wei K, Korsunsky I. Spatial Reconstruction of Interstitial Lung Disease. *In preparation*.
3. Reshef Y, Sood L, ..., **Palshikar MG**, ..., Jonsson AH, Korsunsky I, Raychaudhuri S. Powerful and accurate case-control analysis of spatial molecular data with deep learning-defined tissue microniches. *bioRxiv.* 2024; doi:10.1101/2024.02.07.537149. [[Preprint]](https://www.biorxiv.org/content/10.1101/2024.02.07.537149v2)

Published 
------
1. Millard N, Chen JH, **Palshikar MG**, ..., He J, Wei K, Korsunsky I. Batch correcting single-cell spatial transcriptomics count data with Crescendo improves visualization and detection of spatial gene patterns. *Genome Biol.* 2025;26(1):36. [[DOI]](https://doi.org/10.1186/s13059-025-03479-9)
2. Bolton C, Mahony CB, ..., Garnett J, Raychaudhuri S, Frank MM. Synovial tissue atlas in juvenile idiopathic arthritis reveals pathogenic niches associated with disease severity. *Sci Transl Med.* 2024 Jan 10;16(729):eadt6050. [[PMID:38198634]](https://pubmed.ncbi.nlm.nih.gov/38198634/)
3. Kassis G*, **Palshikar MG***, Hilchey SP, Zand MS, Thakar J. Discrete-state models identify pathway specific B cell states across diseases and infections at single-cell resolution. *J Theor Biol.* 2024 Feb 28;583:111769. [[PMID:38241981]](https://pubmed.ncbi.nlm.nih.gov/38241981/)
4. **Palshikar MG**, Min X, Crystal A, Alexeyenko A, Thakar J. Executable Network Models of Integrated Multiomics Data. *J Proteome Res.* 2023 May 5;22(5):1841-1854. [[PMID:37071060]](https://pubmed.ncbi.nlm.nih.gov/37071060/)
5. **Palshikar MG**, Palli R, ..., Looney RJ, Zand MS, Thakar J. Executable models of immune signaling pathways in HIV-associated atherosclerosis. *npj Syst Biol Appl.* 2022 Sep 21;8(1):35. [[PMID:36131068]](https://pubmed.ncbi.nlm.nih.gov/36131068/)
6. Hilchey SP*, **Palshikar MG***, ..., Bernstein SH, Zand MS, Thakar J. Cyclosporine A Modulates LSP1 Protein Levels in Human B Cells to Attenuate B Cell Migration at Low O₂ Levels. *Life (Basel).* 2022 Aug 12;12(8):1249. [[PMID:36013463]](https://pubmed.ncbi.nlm.nih.gov/36013463/)
7. **Palshikar MG**, Hilchey SP, Zand MS, Thakar J. WikiNetworks: translating manually created biological pathways for topological analysis. *Bioinformatics.* 2022 Feb 1;38(3):869-871. [[PMID:34636843]](https://pubmed.ncbi.nlm.nih.gov/34636843/)
8. Hilchey SP, **Palshikar MG**, ..., Elliott MR, Zand MS, Thakar J. Cyclosporine A Directly Affects Human and Mouse B cell Migration in vitro by Disrupting a HIF1α-Dependent, O₂ Sensing, Molecular Switch. *BMC Immunol.* 2020 Mar 18;21(1):14. [[PMID:32188448]](https://pubmed.ncbi.nlm.nih.gov/32188448/)
9. Palli R, **Palshikar MG**, Thakar J. Executable pathway analysis using ensemble discrete-state modeling for large-scale data. *PLoS Comput Biol.* 2019 Sep 12;15(9):e1007317. [[PMID:31513579]](https://pubmed.ncbi.nlm.nih.gov/31513579/)

Conference Presentations
------
* **Palshikar MG**, Spurrell M, Demaray J, ..., Hacohen N, Chen JH, Korsunsky I. Abstract 5492: Spatial organization and cellular composition of immunity hubs in human colorectal cancer. *Cancer Res.* 2024; 84 (6 Supplement):5492.
* ***Palshikar, M.G.*** Executable models of pathways built using single-cell RNAseq data reveal immune signaling dysregulations in people living with HIV and atherosclerosis American Society for Microbiology (ASM) Eastern New York Fall Symposium, October 1, 2021. American Society for Microbiology (ASM).
* ***Palshikar, M.G.*** WikiNetworks: translating manually created biological pathways for topological analysis. Computational Modeling in Biology Network (COMBINE), October 11-15, 2021.
* ***Palshikar, M.G.*** Executable models of pathways built using single-cell RNAseq data reveal immune cell heterogeneity in people living with HIV and atherosclerosis. Conference on Intelligent Systems for Molecular Biology, July 13-16, 2020. International Society for Computational Biology.

<span style="color:#154360">Teaching and Mentoring</span>
======

* ***Instructor (Spring 2021 & Spring 2022), University of Rochester:***
  - Introduction to Quantitative Biology. 400-level course on quantitative methods and bioinformatics for second-year graduate students at
  University of Rochester Medical Center.
  - Taught a module on unsupervised machine learning methods with applications to the analysis of high-throughput biological
  data. Conducted workshops on applications of these methods in R.
* ***Workshop Teaching Assistant (Fall 2020) and Teaching Assistant (Fall 2018), University of Rochester:***
  - Foundations in Modern Biology - 1. Required 400-levelsurvey course for all incoming graduate students at University of Rochester
  Medical Center
  - Responsible for office hours, designing and conducting weekly group workshops, and grading of exams and assignments
* ***Graduate Student Mentor (Summer 2019 to present), University of Rochester:***
  - Supervised five undergraduate students and one visiting scientist working on network-based methods to analyze large-scale
  omics data
  - Responsible for helping students design methods/algorithms and teaching coding in Python and R
  - Guided students in preparation and presentation of research findings

<span style="color:#154360">Honors and Awards</span>
======

  - Spring 2023: George V. Metzger Award, University of Rochester School of Medicine and Dentistry. Awarded to a Biophysics PhD student based on the quality of the dissertation and the supporting research. Please read more about George Metzger's life [here.](https://www.urmc.rochester.edu/biochemistry-biophysics/awards/metzger-award.aspx)
  - Fall 2017: Graduate Alumni Fellowship Award, University of Rochester School of Medicine and Dentistry. Awarded to an incoming graduate student in the School of Medicine and Dentistry with promise for exceptional
accomplishment in graduate study

<!--  - April 2017: Bioinformatics National Certification Examination. Issued by Department of Biotechnology, Government of India. Ranked 4th nationwide. Eligible for the DBT-BINC junior research fellowship for graduate study in Indian universities.
  - 2017: Prof. Kolaskar Gold Medal, Savitribai Phule Pune University, India. Awarded to the student ranked first in the M.Sc. Bioinformatics program
  - 2015 - 2017: G.N. Ramachandran Fellowship, Bioinformatics Centre, Savitribai Phule Pune University, India. Awarded to the top five students in the M.Sc. Bioinformatics program 

<span style="color:#154360">Undergraduate Research Experience</span>
======

* ***Detection of luxRI homologs in Aeromonas bacteria (Fall 2013)***
  - National Centre for Cell Science Pune, India
  - Advisors: Kamlesh Jangid, PhD & Yogesh Shouche, PhD
  - Project Description: Confirmed the presence of virulence-associated luxRI homologs in 27 environmental isolates of Aeromonas bacteria
using PCR and a biosensor assay
* ***Efficacy Testing of Disinfectants on MS2 Phage (Summer 2013)***
  - Tata Research Design and Development Centre (Process Engineering Lab) Pune, India
  - Advisors: Chetan Malhotra, PhD & Rajshree Patil, PhD
  - Project Description: Determined the minimum exposure times and concentrations at which candidate disinfectants are most effective
against a model virus and hence most suitable for use in a low-cost water purification system under development in the lab -->


<!-- <span style="color:#154360">Technical Skills</span>
======

* ***Programming and Scripting Languages***
  - **Programming Languages**: Python, R, C, Perl.
  - **Markup languages**: LaTeX, Markdown.
  - **Scripting**: SLURM, bash scripting.
  - **Libraries**: fluent in Scikit-Learn, Numpy, Pandas, Jupyter, and standard graph theory, visualization, statistical and bioinformatics packages in Python and R, with some exposure to Tensorflow
  - Some exposure to Java, C++, PHP, Octave, OMP, HTML.
* ***Software Development***
  - Extensive experience in developing and maintaining open-source Python packages on **GitHub** and on the **Python Package Index**.
  - Responsible for writing and maintaining in-house R data analysis scripts across multiple projects. -->
